### ISSUE TITLE
Warning/Error Message Option (Bivariate)

### ISSUE THREAD

**Issue by c-conner** [holoviz/holoviews#4561]
#### Is your feature request related to a problem? Please describe.
Not a problem, just dealing with aesthetics. Basically the issue is the type of message that pops up for plots that will not render when you choose widget values that are the same. I understand from the perspective of a coder looking at error messages, but with end-users who just want to demo or explore their data, seeing a stack trace error would probably be irritating. 

#### Describe the solution you'd like
Instead of an error message popping up when you select the same widget values of a plot (like the Bivariate), could there just be a warning message stating the plot will not render/choose another value?

**Comment by philippjfr** [comment]
Agreed, I've wondered about this myself, statistical plots that can fail to converge such as Distribution, Bivariate and Violin should all handle the failure gracefully and probably just render an empty plot and use the logging module to signal the issue.
